Documentation/v2/metrics.md

```diff
@@ -52,7 +24,7 @@ All these metrics are prefixed with `etcd_http_`
 |--------------------------------|-----------------------------------------------------------------------------------------|--------------------|
 | received_total                 | Total number of events after parsing and auth.                                      | Counter(method)        |
 | failed_total                   | Total number of failed events.                                                      | Counter(method,error)  |
-| successful_duration_second     |  Bucketed handling times of the requests, including raft rounds for writes.          | Histogram(method)      |
+| successful_duration_seconds     |  Bucketed handling times of the requests, including raft rounds for writes.          | Histogram(method)      |
```

Comment:
this one was the old v2 one, which is added by @mwitkow. i changed the new v3 one. i do not want to change the existing one in a significant one. what do you think @mwitkow?